default/stringtables/es.txt

```diff
@@ -1167,11 +1167,45 @@ Escalado de coste de partes
 #*RULE_TEST_STRING_DESC
 #* <not translated 1970-01-01 01:00:00>
 
-#*RULE_STARLANES_EVERYWHERE
-#* <not translated 1970-01-01 01:00:00>
+RULE_STARLANES_EVERYWHERE
+Vías Estelares Completas
```

Comment:
Great, you found a much better translation for starlanes: vías estelares.
Whenever you find mentions to "línea(s) estelare(s)", don't hesitate to replace "línea" by "vía".